### PR TITLE
key_projects: add meson-python

### PR DESCRIPTION
**Pull request by FFY00**
Signed-off-by: Filipe Laíns <lains@riseup.net>

### CODE REVIEW

File: source/key_projects.rst [hunk 1]
```diff
@@ -454,6 +454,22 @@ researchers but has been lacking in maintenance since 2016.
 
 .. _multibuild:
```
Comment (webknjaz) [inline source/key_projects.rst:455]: Hey @FFY00, this label is misplaced — it relates to the previously existing entry and should be right above its title.

File: source/key_projects.rst [hunk 2]
```diff
@@ -454,6 +454,22 @@ researchers but has been lacking in maintenance since 2016.
...
+`GitHub <https://github.com/mesonbuild/meson-python>`__
+
+``meson-python`` is a build backend that uses the Meson_ build system. It enables
+Python package authors to use Meson_ as the build system for their package. It
+supports a wide variety of languages, including C, and is able to fill the needs
+of most complex build configurations.
+
+.. _Meson: https://github.com/mesonbuild/meson
+
+.. _meson-python:
```
Comment (webknjaz) [inline source/key_projects.rst:470]: And this should be above the new section.

Comment (FFY00) [reply source/key_projects.rst:470]: Ah, thanks!

**Review by webknjaz — APPROVED** [review]

**Comment by FFY00** [comment]
Thanks!

**Pull request merged**